control_panel_api/services.py

```diff
@@ -0,0 +1,29 @@
+from django.conf import settings
+
+from . import aws
+
+READ_WRITE = 'readwrite'
+READ_ONLY = 'readonly'
+
+
+def _bucket_name(name):
+    """Prefix the bucket name with environment e.g. dev-james"""
+    return "{}-{}".format(settings.ENV, name)
```

Comment:
I'm wondering how this will work with the model. I feel like we should probably do this in the S3Bucket model somehow? Maybe while saving the record check if it starts with `dev-` already and if not add it behind the scenes?

Reply:
Remember the App has a slug which is regex validated, so as the input comes in it should be all good by the time we pass it into aws

Reply:
You mean the `S3Bucket` model right? Yes, I'm kind of wondering if we should add this prefix there and tweak the regex.

Reply:
As for yesterday's (Slack) discussion, we're keeping it simple and "trust" the user (guided via the UI) to pass an S3 bucket which starts with the right env name.

So basically here we don't prefix the bucket name with `settings.ENV` anymore as the name coming from the request/in the `S3Bucket` model already starts with `settings.ENV` (but we should probably tweak the `S3Bucket` model to validate this.